content/common/language-guide-overview.md

```diff
@@ -0,0 +1,448 @@
+title = "Language Support Overview"
+template = "common_main"
+date = "2022-01-01T00:00:01Z"
+enable_shortcodes = true
+[extra]
+
+---
+
+This page contains information about language support for Spin features:
+
+{{ tabs "sdk-type" }}
+
+{{ startTab "Rust"}}
+
+| Feature | SDK Supported? | Learn More |
+|-----|-----|-------|
+| **Triggers** |
+| HTTP | Supported | [Documentation](https://developer.fermyon.com/spin/http-trigger)<br/> [Code example](https://github.com/fermyon/spin/tree/main/examples/http-rust-outbound-http)<br/>Blog|
+| Redis | Supported | [Documentation](https://developer.fermyon.com/spin/redis-trigger)<br/> [Code example](https://github.com/fermyon/spin/tree/main/examples/redis-rust)<br/>Blog|
+| Custom | Supported | [Documentation](https://developer.fermyon.com/spin/extending-and-embedding)<br/>Code example<br/>Blog|
+| **APIs** |
+| Outbound HTTP | Supported | [Documentation](https://developer.fermyon.com/spin/rust-components.md#sending-outbound-http-requests)<br/>[Code example](https://github.com/fermyon/spin/tree/main/examples/http-rust-outbound-http)<br/>Blog|
+| Key Value Storage | Supported | [Documentation](https://developer.fermyon.com/spin/kv-store.md)<br/>[Code example](https://github.com/fermyon/spin/tree/main/examples/rust-key-value)<br/>Blog|
+| Outbound MySQL | Supported | [Documentation](https://developer.fermyon.com/spin/rdbms-storage#using-mysql-and-postgresql-from-applications)<br/>[Code example](https://github.com/fermyon/spin/tree/main/examples/rust-outbound-mysql)<br/>[Blog](https://www.fermyon.com/blog/building-a-social-app-with-spin-1) |
```

Comment:
I know this is not final wording or anything but I would drop "outbound" from MySQL and PostgreSQL.